spec/rails_helper.rb

```diff
@@ -171,7 +171,7 @@
   # Replace stubbing out CASE_UPLOADS_S3_BUCKET with a test harness of our own
   # making. The problem with stubbing out is that it doesn't work in 'before
   # :all' blocks, which are sometimes needed e.g. ICO appeals which always
-  # require an uploaded document. so can't be instantiate in a 'before :all'
+  # require an uploaded document. so cannot be instantiate in a 'before :all'
```

Comment:
I guess this should be "instantiated" which we might as well fix if we are here... :)

Reply:
👍